packages/core/src/search/parse.ts

```diff
@@ -142,11 +129,12 @@ class SearchParser implements SearchRequest {
         break;
 
       default: {
-        const param = getSearchParameter(this.resourceType, code);
-        if (!param) {
-          throw badRequest('Unknown search parameter: ' + code);
+        const param = globalSchema.types[this.resourceType]?.searchParams?.[code];
```

Comment:
Interesting.  No op needed but I see how this works now.

Reply:
This change is worth a discussion.

Before, this URL parsing code was in the server project, where you could assume that the full FHIR schema was loaded, including all FHIR search parameters.  That meant you could validate search parameters as you were parsing the URL.

After, this URL parsing code is in core, where it might run in the server, or it might run in the browser.  So we're more tolerant of missing search parameters.

At the time that the search is actually executed, we we still validate search parameters, and will throw on unrecognized search param.